autotest/test_budget.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+import pytest
+
+from pynhm.base.budget import Budget
+from pynhm.base.control import Control
+
+# TODO
+# * Test restart more robustly
+# * "Real test cases" in indvidual storage units (e.g. Canopy).
+
+time_dict = {
+    "start_time": np.datetime64("1979-01-03T00:00:00.00"),
+    "end_time": np.datetime64("1979-01-06T00:00:00.00"),
+    "time_step": np.timedelta64(1, "D"),
+}
+
+time_dict["init_time"] = time_dict["start_time"] - time_dict["time_step"]
+
+
+@pytest.fixture(scope="function")
+def control_simple():
+    return Control(**time_dict)
+
+
+def test_budget(control_simple):
```

Comment:
this is just a toy test. budget tests on actual storage units are in the tests of those units.